### PR TITLE
Fix report pathing and re-run tests

### LINKED ISSUE #174: Crash when creating csv file
Since the upgrade from version 1.2.1 to 1.3.3 I can no longer create a csv file
See stack trace:

C:\Users\droussel>neoload report --template builtin:transactions-csv > temp.csv
Traceback (most recent call last):
  File "c:\users\droussel\appdata\local\programs\python\python39\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\users\droussel\appdata\local\programs\python\python39\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\droussel\AppData\Local\Programs\Python\Python39\Scripts\neoload.exe\__main__.py", line 7, in <module>

  File "c:\users\droussel\appdata\local\programs\python\python39\lib\site-packages\click\core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "c:\users\droussel\appdata\local\programs\python\python39\lib\site-packages\click\core.py", line 782, in main
    rv = self.invoke(ctx)
  File "c:\users\droussel\appdata\local\programs\python\python39\lib\site-packages\click\core.py", line 1289, in invoke
    rv.append(sub_ctx.command.invoke(sub_ctx))
  File "c:\users\droussel\appdata\local\programs\python\python39\lib\site-packages\click\core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "c:\users\droussel\appdata\local\programs\python\python39\lib\site-packages\click\core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "c:\users\droussel\appdata\local\programs\python\python39\lib\site-packages\neoload\commands\report.py", line 68, in cli
    model = initialize_model(filter, template)
  File "c:\users\droussel\appdata\local\programs\python\python39\lib\site-packages\neoload\commands\report.py", line 128, in initialize_model
    parse_template_spec(model,filter_spec,template.strip())
  File "c:\users\droussel\appdata\local\programs\python\python39\lib\site-packages\neoload\commands\report.py", line 180, in parse_template_spec
    model["template_text"] = get_builtin_template_transaction_csv()
  File "c:\users\droussel\appdata\local\programs\python\python39\lib\site-packages\neoload\commands\report.py", line 1033, in get_builtin_template_transaction_csv
    return get_resource_as_string('tests/resources/jinja/builtin_transactions_csv.j2').strip()
  File "c:\users\droussel\appdata\local\programs\python\python39\lib\site-packages\neoload\commands\report.py", line 173, in get_resource_as_string
    return pkg_resources.read_text(namespace, file)
  File "c:\users\droussel\appdata\local\programs\python\python39\lib\importlib\resources.py", line 139, in read_text
    with open_text(package, resource, encoding, errors) as fp:
  File "c:\users\droussel\appdata\local\programs\python\python39\lib\importlib\resources.py", line 121, in open_text
    open_binary(package, resource), encoding=encoding, errors=errors)
  File "c:\users\droussel\appdata\local\programs\python\python39\lib\importlib\resources.py", line 87, in open_binary
    resource = _normalize_path(resource)
  File "c:\users\droussel\appdata\local\programs\python\python39\lib\importlib\resources.py", line 62, in _normalize_path
    raise ValueError('{!r} must be only a file name'.format(path))
ValueError: 'tests/resources/jinja/builtin_transactions_csv.j2' must be only a file name
###

### PR DESCRIPTION
**Pull request by paulsbruce**
## What?
Fix issue with path separators as reported by Issue #174 

## Why?
On Windows machines, path separator wasn't being taken into account as a backslash.

## How?
Check all system path separators in relative resource URIs for report templates so that no matter which system it is run on, the resource namespace is derived from the relative path (i.e. tests/resources/jinja/builtin_transactions_csv.j2) properly.

## Testing
Installed on a Win2016 server and verified working now manually. All other unit tests passing. Also, updated two docker-based tests to be run under the same [integration] model as the other tests which rely on Docker, so that unit test suite passes if Docker is not installed or running.

## Additional Notes
This is another thing that would be caught by unit and integration tests if we start using an additional Windows build node in the build and test workflows. Closes #174

### CODE REVIEW

File: neoload/commands/report.py [hunk 1]
```diff
@@ -166,11 +166,20 @@ def get_resource_as_string(relative_path):
...
 
-    path = relative_path.split(os.path.sep)
+
+    path = split_path(relative_path)
     namespace = ".".join(path[:-1])
     file = path[-1]
     logging.debug({'path':path,'namespace':namespace,'file':file})
-    return pkg_resources.read_text(namespace, file)
+    contents = pkg_resources.read_text(namespace, file)
+    logging.debug("get_resource_as_string[contents]: {}".format(contents))
```
Comment (guillaumebert) [inline neoload/commands/report.py:175]: Log the entire file in the logs is not a good idea. This won't help troubleshouting production issues, and the debug logs will be unreadable.
Remove this line

File: tests/commands/docker/test_docker_cleanups.py [hunk 2]
```diff
@@ -11,6 +11,7 @@
 
 @pytest.mark.docker
 @pytest.mark.slow
+@pytest.mark.makelivecalls
```
Comment (guillaumebert) [inline tests/commands/docker/test_docker_cleanups.py:14]: this is not related to "Fix report"

File: tests/commands/docker/test_docker_hooks.py [hunk 3]
```diff
@@ -11,6 +11,7 @@
 
 @pytest.mark.docker
 @pytest.mark.slow
+@pytest.mark.makelivecalls
```
Comment (guillaumebert) [inline tests/commands/docker/test_docker_hooks.py:14]: this is not related to "Fix report"

**Review by guillaumebert — CHANGES_REQUESTED** [review]

File: neoload/commands/report.py [hunk 4]
```diff
@@ -166,11 +166,20 @@ def get_resource_as_string(relative_path):
...
+    return contents
+
+def split_path(path):
+    sep = os.path.sep
```
Comment (stephanemartin) [inline neoload/commands/report.py:179]: Plz use os.path.sep

File: tests/commands/docker/test_docker_hooks.py (hunk 3 above)
Comment (paulsbruce) [reply tests/commands/docker/test_docker_hooks.py:14]: Understood, however this test should have been marked as for integration testing in 1.3.1, and was failing during my local testing of Issue #175 so I added the small change.

File: tests/commands/docker/test_docker_cleanups.py (hunk 2 above)
Comment (paulsbruce) [reply tests/commands/docker/test_docker_cleanups.py:14]: Understood, however this test should have been marked as for integration testing in 1.3.1, and was failing during my local testing of Issue #175 so I added the small change.

**Review by guillaumebert — APPROVED** [review]
Ok for this patch to get files from package on Windows, to resolve customer issue.
There is still a technical debt because the whole test folder should not be included in the released package. To Fix later
